jwql/website/apps/jwql/oauth.py

```diff
@@ -266,7 +271,7 @@ def logout(request):
         Outgoing response sent to the webpage
     """
 
-    response = redirect("/")
+    response = redirect(PREV_PAGE)
```

Comment:
I found that this change didn't let me log out if I hadn't logged in on the same web session. When I clicked the "log out" button, I would just get an error.

But I fixed it by just adding the same lines from `login()` to the `logout()` function, e.g.:
```python
global PREV_PAGE
PREV_PAGE = request.META.get('HTTP_REFERER')
response = redirect(PREV_PAGE)
```
Is this an okay change to make? I don't really understand global variables still.

Reply:
@laurenmarietta Nice catch.  Yeah, this should be fine to do, and I think it makes sense why it broke for you in that situation; in that case the `PREV_PAGE` variable wouldn't have been set in the global scope.